pkg/datapath/linux/config/config.go

```diff
@@ -224,6 +233,23 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		cDefinesMap["ENABLE_PREFILTER"] = "1"
 	}
 
+	if option.Config.SkipSocketLBInPodNamespace {
+		once.Do(func() {
+			found := false
+			if helpers := probes.NewProbeManager().GetHelpers("cgroup_sock_addr"); helpers != nil {
+				if _, ok := helpers["bpf_get_netns_cookie"]; ok {
+					found = true
+				}
+			}
+			if !found {
+				log.Warn("Without network namespace cookie lookup functionality, BPF datapath " +
```

Comment:
I think this log msg can be more specific about implications, i.e. saying that Istio rules will be inactive or so. Also, can we move this check to `daemon/cmd/kube_proxy_replacement.go`?

Reply:
Done.